zmon_worker_monitor/builtins/plugins/cassandra_wrapper.py

```diff
@@ -20,6 +21,10 @@ def configure(self, conf):
         Called after plugin is loaded to pass the [configuration] section in their plugin info file
         :param conf: configuration dictionary
         """
+
+        self._username = conf.get('username', conf.get('user', None))
```

Comment:
This was on purpose, cassandra driver uses username and password and I just wanted to support both, not to confuse but out of convenience